pmoves/design/tests/test_generate.py

```
@@ -0,0 +1,79 @@
+    theme = json.load(open(DESIGN / "themes" / "darkxside-skin.json"))
+    vars_ = resolve_theme(theme, reg)
+    assert vars_["--pm-accent"] == "#E11D48"         # darkxside.color
+    assert vars_["--pm-bg"] == "#0a0608"             # override
+
+
+def test_missing_agent_raises_clear_error():
+    reg = load_registry()
+    bad = {"name": "x", "accents": {"primary": "no-such-agent", "secondary": "darkxside", "signature": "darkxside"}, "overrides": {}}
+    try:
+        resolve_theme(bad, reg)
+        assert False, "expected KeyError"
```

Ruff (0.15.17)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)